### PR TITLE
Kb for modules/auxiliary/scanner/ftp/ftp_login

### PR DESCRIPTION
**Pull request by itsmeroy2012**
### Docs for auxiliary/scanner/ftp/ftp_login

### CODE REVIEW

File: documentation/modules/auxiliary/scanner/ftp/ftp_login.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
+## Description
+
+This module will test FTP logins on a range of machines and report successful logins. If you have loaded a database plugin and connected to a database this module will record successful logins and hosts so you can track your access.
+
+## Vulnerable Application
+
+### Install ftp server on Kali Linux:
+
+1.  ```apt-get install vsftpd```
+2. Allow local users to log in and to allow ftp uploads by editing file /etc/vsftpd.conf uncommenting the following:
```
Comment (h00die) [inline documentation/modules/auxiliary/scanner/ftp/ftp_login.md]: wrap the files with ` so they stand out

File: documentation/modules/auxiliary/scanner/ftp/ftp_login.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+```
+    local_enable=YES
+    write_enable=YES
+    chroot_list_enable=YES
+    chroot_list_file=/etc/vsftpd.chroot_list
+```
+3. Create the file /etc/vsftpd.chroot_list and add the local users you want allow to connect to FTP server. Start service and test connections:
```
Comment (h00die) [inline documentation/modules/auxiliary/scanner/ftp/ftp_login.md]: wrap the files with ` so they stand out

**Comment by h00die** [comment]
awesome, ill take care of this in a few hours

**Comment by itsmeroy2012** [comment]
Thanks @h00die

**Comment by h00die** [comment]
# Release Notes
docs for `auxiliary/scanner/ftp/ftp_login`

**Comment by h00die** [comment]
Thanks as always @itsmeroy2012

**Comment by h00die** [comment]
also part of #8296

**Pull request merged**